nwalsdev/templates/account/login.html

```diff
@@ -1,6 +1,6 @@
 {% extends "site_base.html" %}
 
-{% load account_tags %}
```

Comment:
That's a typo, actually, from when I was experimenting to remove the example.com stuff (couldn't find your answer in the documentation).